app/src/main/res/layout/view_user_profile.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:paddingTop="@dimen/profile_padding_vertical_default"
+    android:paddingStart="@dimen/profile_padding_horizontal_default"
+    android:paddingEnd="@dimen/profile_padding_horizontal_default"
+    android:paddingBottom="@dimen/profile_padding_vertical_default"
+    android:dividerPadding="6dp">
```

Comment:
Dimension resource please. Also, what is this attribute? I haven't used it before.

Reply:
Padding between elements.